articles/the-visual-editor.mdx

```diff
@@ -7,40 +7,84 @@ nav_title: The Visual Editor
 published: true
 image: >-
   https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
-description: Visually navigate and edit the content on your site.
+description: >-
+  Learn about the features of CloudCannon's Visual Editor. This editing
+  interface offers an interactive preview of your site for editing directly onto
+  the page.
+article_category:
+  - Explanation
+article_topic:
+  - Editing interfaces
 tags: []
 related_articles: []
 related_links:
 ---
-Visual Editing empowers your content team to edit on the page itself, and to see results instantly with live previews. Content editors can have clearly defined access to — and control over — their page contents, without needing to worry about anything too technical.
+The Visual Editor is an interactive preview of your site, allowing you to navigate your pages as you would on your live site and edit directly onto the page. This editing interface works for any file that outputs to an HTML page on your site.
 
-The Visual Editor is one of the main ways to manage content in CloudCannon. It mirrors your site itself, so the site's content is the only thing your editors need to know.
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/editing-interfaces/2024-04/CloudCannon-Documentation-Visual-Editor-Data-Panel.png" alt="A screenshot of the Visual Editor,  with the open data panel showing the inputs of a page component." title="The Visual Editor with data panel" type="screenshot"/>
 
-* Navigate to other pages by clicking links on the site.
-* All of your styles and JavaScript remain intact and functional within the Visual Editor.
+<comp.Notice info_type="info">
+  Depending on your SSG and site configuration, you may not have access to the Visual Editor for all files. An editing interface may not be available because:
 
-## Editing markup
+  * Some SSGs are incompatible with the Visual Editor (e.g., Docusaurus, MkDocs).
```

Comment:
Is there evidence we can provide for this statement? I imagine as a user of those SSGs someone may want to know more. They should have access to data bindings right?